slick/src/main/scala/slick/lifted/Query.scala

```diff
@@ -210,10 +210,6 @@ sealed abstract class Query[+E, U, C[_]] extends QueryBase[C[U]] { self =>
   /** The total number of elements (i.e. rows). */
   def size = length
 
-  /** The number of distinct elements of the query. */
-  @deprecated("Use `length` on `distinct` or `distinctOn` instead of `countDistinct`", "3.2")
-  def countDistinct: Rep[Int] = Library.CountDistinct.column(toNode)
```

Comment:
please don't remove until fixed 🙏 

- https://github.com/slick/slick/issues/1760

Reply:
I'm on it